cmake/ciToolsUpdate.cmake

```diff
@@ -1,21 +1,45 @@
 set(cmake_version $ENV{CMAKE_VERSION})
 set(ninja_version $ENV{NINJA_VERSION})
 
+set(cmake_arch "x86_64")
+
 if(RUNNER_OS STREQUAL "Windows")
   set(ninja_suffix "win.zip")
-  set(cmake_suffix "win64-x64.zip")
-  set(cmake_dir "cmake-${cmake_version}-win64-x64/bin")
-elseif(RUNNER_OS STREQUAL "Linux")
-  set(ninja_suffix "linux.zip")
-  set(cmake_suffix "Linux-x86_64.tar.gz")
-  set(cmake_dir "cmake-${cmake_version}-Linux-x86_64/bin")
-elseif(RUNNER_OS STREQUAL "macOS")
-  set(ninja_suffix "mac.zip")
-  set(cmake_suffix "Darwin-x86_64.tar.gz")
-  set(cmake_dir "cmake-${cmake_version}-Darwin-x86_64/CMake.app/Contents/bin")
+  set(cmake_ext "zip")
+  if("${cmake_version}" VERSION_GREATER_EQUAL "3.20.0")
```

Comment:
Are we sure we want to distinguish 64/32 but based on the cmake version?

Reply:
I'm confused. I'm always using x86_64. CMake changed their platform naming convention with 3.20, which is all that is happening with the version comparison, see https://github.com/Kitware/CMake/releases/tag/v3.19.0 vs https://github.com/Kitware/CMake/releases/tag/v3.20.0.